config/locales/fr.yml

```diff
@@ -599,3 +599,5 @@ fr:
   price_graph: "Légende détail du prix"
   included_tax: "Inclut TVA"
   remove_tax: "Retirer TVA"
+  calculator_weight_per_kg: "Poids (par kg)"
```

Comment:
No. Transifex is free for open source projects. And you can manage the locales yourself as well. But it will be really difficult to do it manually. You need some kind of tool to find out which bits need translating.